app/views/shared/_list_of_things.html.erb

```diff
@@ -10,16 +10,18 @@
         <dl class="dl-horizontal">
             <dt>Study details</dt>
             <dd>
-                <a href="/search?methodology=cohort%20study"><%= study.study_type.name %></a>
+                <%= link_to study.study_type.name, search_path(study_type: study.study_type.name.downcase) %>
```

Comment:
Since the search SQL does down-casing anyway, I think it would be nicer to leave this in the original capitalisation, for when we set titles from it and suchlike